src/store.rs

```diff
@@ -29,6 +29,19 @@ impl Store {
         let key = format_session_key(session_id);
         self.client.hgetall(key.as_str()).map_err(stringify_err)
     }
+
+    pub fn store_cache(&self, url: &str, data: &str)
```

Comment:
I'm not sure this is the right abstraction for this. Maybe we can add a separate `StoreCache` trait that takes a store, and then store in there? I think it's also better not to instantiate a new `Http::Client` for every request. The `StoreCache` should probably have a single method that takes `type` (so we can disambiguate different types of cached data, instead of prefixing them all with `cache:`), `key`, `session` (an `Http::Client`), and the URL to get, and returns the value.